src/patternfly/components/Wizard/examples/wizard-example.hbs

```diff
@@ -0,0 +1,86 @@
+{{#> wizard id="wizard-simple-example"}}
+  {{#> background-image}}{{/background-image}}
+  {{#> wizard-header}}
+    {{#> button button--modifier="pf-m-plain pf-c-wizard__close" button--attribute='aria-list="Close"'}}
+      {{> button-icon button-icon--type="times"}}
+    {{/button}}
+    {{#> title title--modifier="pf-m-3xl pf-c-wizard__title"}}Wizard Title{{/title}}
+    {{#> wizard-description}}
+      here is where the description goes
+    {{/wizard-description}}
+  {{/wizard-header}}
+  {{#> wizard-steps}}
+    {{#> wizard-toggle}}
+     {{#> wizard-toggle-list}}
+        {{#> wizard-toggle-list-item}}
+          {{#> wizard-toggle-num}}2{{/wizard-toggle-num}}
+          Configuration
+          {{> wizard-toggle-divider}}
+        {{/wizard-toggle-list-item}}
+        {{#> wizard-toggle-list-item}}
+          Substep B
+        {{/wizard-toggle-list-item}}
+      {{/wizard-toggle-list}}
+      {{> wizard-toggle-icon}}
+    {{/wizard-toggle}}
+    {{#> wizard-nav}}
+      {{#> wizard-nav-list}}
+        {{#> wizard-nav-item}}
+          {{#> wizard-nav-link}}
+            Information
+          {{/wizard-nav-link}}
+        {{/wizard-nav-item}}
+        {{#> wizard-nav-item}}
+          {{#> wizard-nav-link wizard-nav-link--modifier="pf-m-current"}}
+            Configuration
+          {{/wizard-nav-link}}
+          {{#> wizard-nav-list}}
+            {{#> wizard-nav-item}}
+              {{#> wizard-nav-link}}
+                 Substep A
+              {{/wizard-nav-link}}
+            {{/wizard-nav-item}}
+            {{#> wizard-nav-item}}
+              {{#> wizard-nav-link wizard-nav-link--modifier="pf-m-current"}}
```

Comment:
Similar to our Nav component, we should include `aria-current="page` for the current selected page.